cvat/apps/annotation/README.md

```diff
@@ -150,3 +150,173 @@ It allows to download and upload annotations in different formats and easily add
+  ├── ...
+  └──obj.names
+  ```
+  Each `*.txt` file contains annotations for a frame with the same name as annotation file. Note: it should be possible to match the CVAT frame(imagename) and annotation filename. There are 2 options:
+  1. full match between image name and filename form annotation `*.txt` file (in case of a task was created from images or archive of images).
+  2. match by frame number (if CVAT cannot match by name). If name of the *.txt annotation file contains a number CVAT will interpret it as frame number.
+
+- supported shapes: Rectangles
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+### [MS COCO Object Detection](http://cocodataset.org/#format-data)
+#### Dumper description
+- downloaded file: single unpacked `json`. Detailed description of the MS COCO format can be found [here](http://cocodataset.org/#format-data)
+- supported shapes - Polygons, Rectangles (interpreted as polygons)
+
+#### Loader description
+- uploaded file: single unpacked `*.json`.
+- supported shapes: Polygons
+- additional comments: the CVAT task should be created with the full label set that may be in the annotation files
+
+#### How to create a task from PMS COCO dataset
+1. Download the [MS COCO dataset](http://cocodataset.org/#download). For example [2017 Val images](http://images.cocodataset.org/zips/val2017.zip) and [2017 Train/Val annotations](http://images.cocodataset.org/annotations/annotations_trainval2017.zip).
+1. Create a CVAT task with the following labels:
+   ```
+   "person" "bicycle" "car" "motorcycle" "airplane" "bus" "train" "truck" "boat" "traffic light" "fire hydrant" "stop sign" "parking meter" "bench" "bird" "cat" "dog" "horse" "sheep" "cow" "elephant" "bear" "zebra" "giraffe" "backpack" "umbrella" "handbag" "tie" "suitcase" "frisbee" "skis" "snowboard" "sports ball" "kite" "baseball bat" "baseball glove" "skateboard" "surfboard" "tennis racket" "bottle" "wine glass" "cup" "fork" "knife" "spoon" "bowl" "banana" "apple" "sandwich" "orange" "broccoli" "carrot" "hot dog" "pizza" "donut" "cake" "chair" "couch" "potted plant" "bed" "dining table" "toilet" "tv" "laptop" "mouse" "remote" "keyboard" "cell phone" "microwave" "oven" "toaster" "sink" "refrigerator" "book" "clock" "vase" "scissors" "teddy bear" "hair drier" "toothbrush"
+   ```
+
+   Select val2017.zip as data (See [Creating an annotation task](cvat/apps/documentation/user_guide.md#creating-an-annotation-task) guide for details)
+1. unpack annotations_trainval2017.zip
+1. click `Upload annotation` button, choose `COCO JSON 1.0` and select `instances_val2017.json.json` annotation file. It may take some time.
+
+### [TFRecord](https://www.tensorflow.org/tutorials/load_data/tf_records)
+TFRecord is a very flexible format, but we try to correspond the format that used in [TF object detection](https://github.com/tensorflow/models/tree/master/research/object_detection) with minimal modifications.
```

Comment:
Need an example for TFRecord. Use https://github.com/tensorflow/models/blob/master/research/object_detection/dataset_tools/create_pascal_tf_record.py script to write instructions.